src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Default {
```

Comment:
The class IsoscelesTrapezoid should not extend Default. Consider extending a more appropriate class or implementing an interface that matches the intended behavior.